plugins/uniques.user.js

```diff
@@ -367,6 +373,111 @@ window.plugin.uniques.loadLocal = function(name) {
 	}
 }
 
+window.plugin.uniques.portalShouldBeHidden = function(portal) {
+  var uniqueInfo = window.plugin.uniques.uniques[portal.options.guid];
+  if(uniqueInfo) {
+    var hideAsVisited = window.plugin.uniques.hideVisitedPortals && uniqueInfo.visited;
+    var hideAsCaptured = window.plugin.uniques.hideCapturedPortals && uniqueInfo.captured;
+    return hideAsVisited || hideAsCaptured;
+  }
+  return false;
+}
+
+window.plugin.uniques.updateOtherPluginLabels = function() {
+  if (window.plugin.portalNames)
+    window.plugin.portalNames.updatePortalLabels();
```

Comment:
It's not best idea to call foreign plugins functions directly.
AFAIK, this patch was made before needed hooks were implemented (https://github.com/iitc-project/ingress-intel-total-conversion/pull/977)